test/features/user_records_an_application_test.rb

```diff
@@ -99,6 +99,8 @@ def fill_in_all_application_form_fields
     fill_in "application_location", with: "Chicago, IL"
     fill_in "application_url", with: "http://basecamp.com/jobs"
     fill_in "application_applied_on", with: Date.today
+    fill_in "application_position", with: "Rails Dev"
```

Comment:
There are no assertions in the feature test that verify that the notes or position have been added.